api/bin/chainflip-ingress-egress-tracker/src/witnessing/state_chain.rs

```diff
@@ -84,6 +104,22 @@ enum WitnessInformation {
 		tx_out_id: TransactionId,
 		tx_ref: TransactionRef,
 	},
+	VaultDeposit {
+		#[serde(skip_serializing)]
+		tx_id: TransactionInIdForAnyChain,
+		deposit_chain_block_height: <AnyChain as Chain>::ChainBlockNumber,
+		input_asset: cf_chains::assets::any::Asset,
+		output_asset: cf_chains::assets::any::Asset,
+		amount: NumberOrHex,
+		destination_address: EncodedAddress,
+		deposit_metadata: Option<CcmDepositMetadata>,
+		deposit_details: Option<DepositDetails>,
+		broker_fee: Beneficiary<AccountId32>,
+		affiliate_fees: Affiliates<AffiliateShortId>,
```

Comment:
would be great to have the account ids instead of the short ids here

Reply:
This is turning out to be a non-trivial change. Can we leave this for a later update?

Reply:
unfortunately we will need to do a separate rpc call to handle the short ids on our side, so i would really prefer to have them in redis

Reply:
its fine for me to do that in a follow up pr obv, but that follow up should land before we start using it 🙂

Reply:
I managed to get it working. Not the nicest, but it will do. At least we are one step closer to storing all swap data in redis. Shouldn't be too hard now to fetch the deposit channel data and get it in the db.